django/contrib/gis/db/models/functions.py

```diff
@@ -193,9 +193,11 @@ def __init__(self, expression, relative=False, precision=8, **extra):
         super(AsSVG, self).__init__(*expressions, **extra)
 
 
-class BoundingCircle(GeoFunc):
-    def __init__(self, expression, num_seg=48, **extra):
-        super(BoundingCircle, self).__init__(*[expression, num_seg], **extra)
+class BoundingCircle(OracleToleranceMixin, GeoFunc):
+    def as_postgresql(self, compiler, connection):
+        clone = self.copy()
+        clone.set_source_expressions(self.get_source_expressions() + [Value(self.extra.get('num_set', 48))])
```

Comment:
`num_set`looks like a typo for `num_seg`? Is testing of that parameter inadequate to detect the regression? If so, would be nice to add a test for it in a separate PR.